includes/class-wc-stripe-intent-controller.php

```diff
@@ -720,15 +720,15 @@ public function create_and_confirm_payment_intent( $payment_information ) {
 		$request = array_merge(
 			$request,
 			[
-				'amount'               => $payment_information['amount'],
-				'confirm'              => 'true',
-				'currency'             => $payment_information['currency'],
-				'customer'             => $payment_information['customer'],
+				'amount'                    => $payment_information['amount'],
+				'confirm'                   => 'true',
+				'currency'                  => $payment_information['currency'],
+				'customer'                  => $payment_information['customer'],
 				/* translators: 1) blog name 2) order number */
-				'description'          => sprintf( __( '%1$s - Order %2$s', 'woocommerce-gateway-stripe' ), wp_specialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES ), $order->get_order_number() ),
-				'metadata'             => $payment_information['metadata'],
-				'payment_method_types' => $payment_method_types,
```

Comment:
Hey @wjrosa ! Thanks for addressing this issue 🙂 

Curious why removing the `payment_method_types` property here. Mind providing a bit more context behind this change?

Reply:
Hi Danae! Sure, it is just that it is not compatible. If we keep both we get the following error from Stripe:
```
You may only specify one of these parameters: ["automatic_payment_methods", "payment_method_types"].
```